kubejs/server_scripts/base/recipes/minecraft/shaped.js

```diff
@@ -0,0 +1,43 @@
+ServerEvents.recipes(event => {
+    const id_prefex = 'enigmatica:base/minecraft/shaped/';
+
+    const recipes = [
+        {
+            output: '16x minecraft:stick',
+            pattern: ['A', 'A'],
+            key: {
+                A: '#minecraft:logs'
+            },
+            id: `${id_prefex}stick_16`
+        },
+        {
+            output: '4x minecraft:chest',
+            pattern: ['SSS', 'S S', 'SSS'],
+            key: {
+                S: '#minecraft:logs'
+            },
+            id: `${id_prefex}chest_4`
+        },
+        {
+            output: 'minecraft:chest',
```

Comment:
no polymorph yet, so this might just override recipes

Reply:
I'll just remove them since we have Every Compat. Shouldn't ever need to do this kind of thing for wood objects again.

Will make a note to make a proper wood registry again later so we can have 8x logs to 4x chests for all chests/logs in the future.